documentation.md

```diff
@@ -7,50 +7,50 @@ title: Documentation
 
 ## The essentials
 
-*   [the FAQ](http://haskell.org/haskellwiki/Xmonad/Frequently_asked_questions)
-*   [the wiki](http://haskell.org/haskellwiki/Xmonad)
+*   [the FAQ](https://wiki.haskell.org/Xmonad/Frequently_asked_questions)
+*   [the wiki](https://wiki.haskell.org/Xmonad)
 *   the documentation:
-    *    [xmonad](http://hackage.haskell.org/package/xmonad)
-    *    [xmonad-contrib](http://hackage.haskell.org/package/xmonad-contrib)
+    *    [xmonad](https://hackage.haskell.org/package/xmonad)
+    *    [xmonad-contrib](https://hackage.haskell.org/package/xmonad-contrib)
 
 ## Getting started
 
 * [about](about.html) – an overview of xmonad features
 * [guided tour](tour.html) – a walkthrough of the basic functionality
-* [step-by-step](http://haskell.org/haskellwiki/Xmonad/Config_archive/John_Goerzen's_Configuration) – guide to configuring xmonad
-* [cheatsheet](http://haskell.org/haskellwiki/Image:Xmbindings.png) – an overview of the keybindings
+* [step-by-step](https://wiki.haskell.org/Xmonad/Config_archive/John_Goerzen's_Configuration) – guide to configuring xmonad
+* [cheatsheet](https://wiki.haskell.org/File:Xmbindings.png) – an overview of the keybindings
```

Comment:
It may be worth to link to the [image](https://wiki.haskell.org/wikiupload/b/b8/Xmbindings.png) directly.  Though this is probably not _that_ important since we want to move everything to the repo sooner or later anyways.